.bazelrc

```diff
@@ -148,3 +148,8 @@ build --flag_alias=ckms_cert_endorsement=//sw/device/silicon_creator/manuf/skus/
 
 # cquery output option.
 cquery --output=files
+
+# Allow the host's `/tmp` directory to be accessed by rules.
+# From Bazel 7 onwards, the sandbox mounts over the top of `/tmp` causing it to not be visible.
+# Some contributors mount EDA tools in `/tmp` which can cause `PATH` problems.
+build --sandbox_add_mount_pair=/tmp
```

Comment:
Obliterating the hermetic tmp doesn't seem like the right approach here. Do we really want to make all users lose the benefits of hermetic tmp?

Should we ask contributors to add their specific subdirectory mounts in a `.bazelrc-site` instead? (not `/tmp`, but `/tmp/tools` or whatever is specific to their site)

Reply:
Hm... but the problem with that POV is that it preferences i
users who don't foolishly add executable binaries to /tmp (security risks). The users who follow the guidance for tmpfs can't readily undo this to get hermetic tmp back. In effect, this change harms usability for everyone else.